fetch_crypto_prices.py

```diff
@@ -2,24 +2,69 @@
 import json
 import os
 from datetime import datetime
+import logging
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
How important is it to use the official `logging` library considering this is a very simple python script to fetch & store prices in a json file using github actions? As long as it shows up in the workflow's log I'm happy